README.md

```diff
@@ -82,7 +82,7 @@ Configure.With(someContainerAdapter)
 // have IBus injected in application services for the duration of the application lifetime    
 
 // let the container dispose the bus when your application exits
-myFavoriteIocContainer.Dispose();
+someContainerAdapter.Dispose();
```

Comment:
Hi Linda,

Thanks for the PR 😄 

I understand why this can be unclear at first, but it is actually not an error.

The usage pattern is basically like this:
```csharp
using(var container = new YourIoCContainer())
{
    Configure.With(new TheContainerAdapter(container))
        .(...)
        .Start();

    // application runs now
    Thread.Sleep(10000);

    // container disposes its contents, including the bus
}
```
which means that the container adapter is simply used to adapt it to be used by Rebus to look up message handlers

Reply:
I am just learning this seams confusing to me.    But i will give it a go :)

Reply:
cool!

The thing is: IoC containers (at least those that understand how they ought to work 😉 ) are good at functioning as a sort of "application container", i.e. they can contain everything that it takes to make out an app, and then they can dispose everything that needs to be disposed, when the container itself is disposed.

Therefore it makes sense to use an IoC container in conjunction with Rebus by carrying out the following steps:

1. Create IoC container
1. Configure Rebus to use IoC container
1. Let application run
1. Dispose IoC container when the application shuts down

I hope that it makes sense now 😄